README.rst

```diff
@@ -5,12 +5,34 @@ This is a command-line interface for interacting with connexions content. The to
 
 This software requires:
 
- - Python >= 3.5
- - JRE >= 6
+- Python >= 3.5
+- JRE >= 6
+
+Install
+=======
+
+1. Install `git`
+1. Install `python3` (on OSX you can run `brew install python3`)
+1. Install `virtualenv`
+1. Download the source `git clone https://github.com/Connexions/nebuchadnezzar`
+1. `cd /path/to/nebuchadnezzar`
+1. Initialize the python virtual environment:
+  1. `virtualenv ./venv/ --python=python3.5`
+  1. `source ./venv/bin/activate`
+  1. `python setup.py develop`
```

Comment:
hmm, I still get an install error when running those steps:

```
pip3 install nebuchadnezzar
Collecting nebuchadnezzar
  Could not find a version that satisfies the requirement nebuchadnezzar (from versions: )
No matching distribution found for nebuchadnezzar
You are using pip version 8.1.2, however version 9.0.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
byzero:nebuchadnezzar phil$ pip install --upgrade pip
Collecting pip
  Using cached pip-9.0.1-py2.py3-none-any.whl
Installing collected packages: pip
  Found existing installation: pip 8.1.2
    Uninstalling pip-8.1.2:
      Successfully uninstalled pip-8.1.2
Successfully installed pip-9.0.1
byzero:nebuchadnezzar phil$ pip3 install nebuchadnezzar
Collecting nebuchadnezzar
  Could not find a version that satisfies the requirement nebuchadnezzar (from versions: )
No matching distribution found for nebuchadnezzar
You are using pip version 8.1.2, however version 9.0.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
```

Maybe the package is not available yet?